core/chains/evm/config/toml/defaults/Holesky_testnet.toml

```diff
@@ -0,0 +1,22 @@
+ChainID = '17000'
+FinalityTagEnabled = true
+
+[GasEstimator]
+EIP1559DynamicFees = true
+PriceMax = '1000 gwei'
+LimitDefault = 8000000
```

Comment:
these are ccip specific configs, if moved to core, it will override with this value, not sure what config we used in the infra cluster